src/pages/signup-affiliates-details/common/_utility.js

```diff
@@ -0,0 +1,40 @@
+import validation from './_validation'
+
+export const getSignupAffiliateValue = (input) => {
+    // TODO Implement future form data transfering
+    return input
+}
+
+export const resetSignupAffiliateDetails = (values, is_input_country) => {
```

Comment:
Can we simplify these code to something like this?

```
const inputFields = [firstName,lastName,date,country,address];

inputFields.map((item)=> {
	return errors.item = validation[item](values[item])
})
```

Reply:
Thanks, refactored here.